cookbook/docs/integrations.rst

```diff
@@ -55,9 +67,28 @@ External Service Backend Plugins
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 #. AWS Sagemaker Training
 #. AWS Batch
-
 #. Qubole Hive
 
+Enabling Backend Plugins
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To enable a backend plugin you have to add the ``ID`` of the plugin to the enabled plugins list. The ``enabled-plugins`` is available under the ``tasks > task-plugins`` section of FlytePropeller's configuration.
+The `plugin configuration structure is defined here <https://pkg.go.dev/github.com/flyteorg/flytepropeller@v0.6.1/pkg/controller/nodes/task/config#TaskPluginConfig>`_. An example of the config follows,
+
+.. rli:: https://raw.githubusercontent.com/flyteorg/flyte/master/kustomize/overlays/sandbox/flyte/config/propeller/enabled_plugins.yaml
+    :language: yaml
+
+Finding the ``ID`` of the Backend Plugin
+""""""""""""""""""""""""""""""""""""""""
+This is a little tricky since you have to look at the source code of the plugin to figure out the ``ID``. In the case of Spark, for example, the value of ``ID`` is `used <https://github.com/flyteorg/flyteplugins/blob/v0.5.25/go/tasks/plugins/k8s/spark/spark.go#L424>`_ here, defined as `spark <https://github.com/flyteorg/flyteplugins/blob/v0.5.25/go/tasks/plugins/k8s/spark/spark.go#L41>`_.
+
+Enabling a Specific Backend Plugin in Your Own Kustomize Generator
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+Flyte uses Kustomize to generate the the deployment configuration which can be leveraged to `kustomize your own deployment <https://github.com/flyteorg/flyte/tree/master/kustomize>`_.
+
+We will soon be supporting helm or a better deployment model - See issue :issue:`299`.
```

Comment:
Is this still valid?

Reply:
yeah, looks like the issue's still open
https://github.com/flyteorg/flyte/issues/299

Reply:
thanks!